peakina/datasource.py

```diff
@@ -111,7 +112,11 @@ def _get_single_df(
         if kwargs.get("sheet_name", NOTSET) is None:
             for sheet_name, _df in df.items():
                 _df["__sheet__"] = sheet_name
-            df = pd.concat(df.values(), sort=False)
+
+            if type(df.values) == numpy.ndarray:
+                df = pd.concat([df], sort=False)
+            else:
+                df = pd.concat(df.values(), sort=False)
```

Comment:
Because am having as output a strange `numpy.ndarray` type for the `df` here... still trying to figured out why 🤔